service/create/service.go

```diff
@@ -492,10 +519,21 @@ func (s *Service) runMachine(input runMachineInput) error {
 			},
 		},
 	}); err != nil {
-		return microerror.MaskAny(err)
+		return nil, microerror.MaskAny(err)
 	}
 
 	s.logger.Log("info", fmt.Sprintf("instance '%s' tagged", input.name))
+	instance := reservation.Instances[0]
 
-	return nil
+	return instance, nil
```

Comment:
I agree with your comment, and I thought this change addresses it: we make this change, and then we don't have to make any more, since we now return a pointer to the instance.

Or am I missing something?